sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -139,21 +140,22 @@ static void _on_sock_evt(sock_udp_t *sock, sock_async_flags_t type, void *arg)
 {
     (void)arg;
     sock_udp_ep_t remote;
+    sock_udp_aux_rx_t aux;
 
     if (type & SOCK_ASYNC_MSG_RECV) {
-        ssize_t res = sock_udp_recv(sock, _listen_buf, sizeof(_listen_buf),
-                                    0, &remote);
+        ssize_t res = sock_udp_recv_aux(sock, _listen_buf, sizeof(_listen_buf),
+                                    0, &remote, &aux);
         if (res <= 0) {
```

Comment:
Is changing the signature fine or should a separate function be defined for the use of the pseudo-module?